guide/app_accuracy.tex

```diff
@@ -33,9 +33,9 @@ \chapter{Accuracy}
 the Solar gravity field, may still be required for critical
 observations of stellar occultations by asteroids, grazing
 occultations of stars by the Moon or similar things. Stellarium does
-not compute these effects, and is definitely not capable for detailed
+not compute these effects, and is definitely not capable of detailed
 planning of your spacecraft missions to the planets.
-There are more specialized programs for these topics, please use them.
+There are more specialized programs for these topics. Please use them.
 
```

Comment:
For a particular large group of readers, somebody recommended using short sentences. I wanted to be understood. ;-)